packages/botonic-cli/src/commands/train.ts

```diff
@@ -1,44 +1,104 @@
 import { Command, flags } from '@oclif/command'
 import colors from 'colors'
-import * as path from 'path'
+import { existsSync } from 'fs'
```

Comment:
use pathExists from `util/file-system` to not introduce more noise. If it's not present in your current code, maybe you will need to rebase with master